### DISCUSSION TITLE
Vieb 9.3.0

### DISCUSSION THREAD

**Discussion by Jelmerro** [Jelmerro/Vieb#424] [Announcements]
### Added

- Setting "reloadtaboncrash" to automatically recreate a crash tab with the last known url
- Indicator for unresponsive tabs in the form of a small red border on the left of the unresponsive tab
- Setting "userscript" to run custom JavaScript on pages, optionally using the GreaseMonkey API

### Changed

- Modifiers/shifted keys for searching, marks and follow mode now work more reliable
- Tabs will no longer refuse to navigate when there is a beforeunload listener that calls preventDefault
- When the current tab is crashed, you can no longer enter irrelevant modes in it

### Fixed

- Initial tab load issues on some devices for specific pages
- Split state not being kept when switching container name based on containernames setting
- Multiple useragents being set sometimes causing duplicate page loads
- BatteryManager errors for insecure pages, since it's an HTTPS-only API
- Follow mode and other main process calls not succeeding when there are crashed tabs present

### Versions

- Electron 21.1.0 (was 21.0.1)
- Chromium 106.0.5249.91 (was 106.0.5249.61)

For more details see the [CHANGELOG](https://github.com/Jelmerro/Vieb/blob/master/CHANGELOG.md).
For startup help see the [frequently asked questions](https://github.com/Jelmerro/Vieb/blob/master/FAQ.md).

<hr /><em>This discussion was created from the release <a href='https://github.com/Jelmerro/Vieb/releases/tag/9.3.0'>Vieb 9.3.0</a>.</em>

**Comment by Marin-Kitagawa** [comment]
Hey @Jelmerro will you be adding the hinting mode (which is done using `;` key) of Qutebrowser anytime soon? If it is already implemented, could you please pardon my ignorance and say how to enable it?

**Reply by Jelmerro** [reply]
I havent' used qutebrowser for more than 15 minutes at most, what exactly does the hinting mode do?

**Reply by Marin-Kitagawa** [reply]
When you press `;` it will hint the below things based on the next keystroke

![image](https://user-images.githubusercontent.com/49131888/196223900-70e3cd0a-d759-4bb0-bd06-35e764ac0c20.png)

**Reply by Jelmerro** [reply]
It's not exactly the same, but instead of hinting, Vieb has follow mode, which can be combined with pointer mode to hover or interact with images/videos and such. The elements shown in follow mode are controlled with the `followelement` setting. Not all of the cases above are possible with it, and for interacting with tabs, you can use the buffer command, like in Vim.
